Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/common/CommonConst.java

```diff
@@ -72,4 +72,8 @@
     public static final String DISABLE_SSL_CERTIFICATE_VALIDATION = "false";
 
     public static final String CosmosServerlessToolWindowIconName= "SparkSubmissionToolWindow";
+
+    public static final String[] AZURE_LOGIN_HOSTS = new String[] {
+            "login.windows.net"
```

Comment:
Maybe we can put it in much related class `com.microsoft.azure.hdinsight.sdk.common.CommonConstant`. Please check `CommonConstant.authority`

Reply:
Good suggestion!

Reply:
The `com.microsoft.azure.hdinsight.sdk.common.CommonConstant` is in different project, which has namespace conflict here...